core/quantity/src/test/Parsing.test.ts

```diff
@@ -107,6 +107,55 @@ describe("Parsing tests:", () => {
     }
   });
 
+  it("Generate Parse Tokens given different decimal and thousand separators", async () => {
+    const testStrings = ["", "0/1", "1/0", "1/2", "-1/2", "+1/2", "2 /", "1,616252eggs", "1,616252E-35eggs", "-1,616252E-35eggs", "756345,345", "12.345,345", "3,6252e3 Miles", "-1 1/2 FT", "+1 1/2 FT", "-135°11'30,5\"", "-2FT 6IN", "135°11'30,5\"", "2FT 6IN", "1 1/2 FT"];
```

Comment:
I would probably narrow down the scenarios to only those that use different decimal and thousands separators to keep the tests more compact or make room for additional new test cases specific to the separator problem

Reply:
got it, I've taken a look through the tests and removed ones that don't apply separators at all